### ISSUE TITLE
Problems with the binary file hwm123114.bin

### ISSUE THREAD

**Issue by lauratrain99** [rilma/pyHWM14#8]
Hi! I followed the installation available in the Readme and got the **pyhwm2014** module installed on my computer. When I import **pyhwm2014** on Python (version 3.8), no error appears. However, following the testing you provide in the Readme (coverage run tests/test.py -v), I get an error saying: 
![image](https://user-images.githubusercontent.com/60850007/104014503-6aef5e00-51b3-11eb-9e45-1170e3fcf46d.png)

However, the file **hwm123114.bin** can be found in the _data_ folder.

I also tried the other example in the Readme running it from Python.
![image](https://user-images.githubusercontent.com/60850007/104015044-5f506700-51b4-11eb-955a-d56bbdc2a4bf.png)

I get a similar problem related to the file **hwm123114.bin**, now saying this:
![image](https://user-images.githubusercontent.com/60850007/104014984-3b8d2100-51b4-11eb-9131-ea5bed0a567e.png)

It looks like there is a problem with the **hw14.f90** source file. I tried with the last release (2017), the master branch and the version 0.02 and the same happens.

**Comment by mee5-pdx** [comment]
Hi! I also had trouble with this. 

However, when I tried installing from the "Installation From Source Code" directions, it worked! I previously tried the pip installation from PyPi, which appeared to work at first, but ended up causing the error with the bin file when I ran it with code. 

Note: I had problems with the last command, `pip install -e . --process-dependency-links`, and used `pip install -e ./` instead, which happened to work.
